### ISSUE TITLE
Keybase cli support

### ISSUE THREAD

**Issue by ifuchs** [ish-app/ish#1216]
Any chance of adding the Keybase cli command?

**Comment by ifuchs** [comment]
Tried installing via app add:

apk add keybase-client
(1/1) Installing keybase-client (5.2.1-r1)
Executing busybox-1.31.1-r19.trigger
OK: 135 MiB in 63 packages
Iras-iPad-Pro-3:~# keybase
Error relocating /usr/bin/keybase: __dlsym_time64: symbol not found
Error relocating /usr/bin/keybase: __nanosleep_time64: symbol not found

**Comment by tbodt** [comment]
This error usually means you're installing different packages from different alpine versions. Try getting everything on one version. This can be done by editing your /etc/apk/repositories to remove other versions and running apk upgrade.

**Comment by ifuchs** [comment]
This is what my /etc/api/repositories looks like:

file:///ish/apk/main
file:///ish/apk/community
file:///ish/apk/community
http://dl-cdn.alpinelinux.org/alpine/v3.13/main
http://dl-cdn.alpinelinux.org/alpine/v3.13/community
http://dl-cdn.alpinelinux.org/alpine/edge/main
http://dl-cdn.alpinelinux.org/alpine/edge/community
http://dl-cdn.alpinelinux.org/alpine/edge/testing
http://dl-cdn.alpinelinux.org/alpine/latest-stable/main
http://dl-cdn.alpinelinux.org/alpine/latest-stable/community

What do you recommend?

**Comment by tbodt** [comment]
Make up your mind :)

Choose one of /ish/apk, v3.13, edge, or latest-stable. In this case edge is really the only choice, since keybase-client only exists in edge.

**Comment by ifuchs** [comment]
Thanks but I eliminated all but the edge repositories and I get the same error after adding the keybase-client

**Comment by tbodt** [comment]
Did you run apk upgrade?

**Comment by ifuchs** [comment]
Yes I ran api update and api fix did not help. I have no idea if the
problem is related but if I do: api list musl , I get:

apk list musl
musl-1.1.24-r9 x86 {musl} (MIT) [installed]
musl-1.2.2_pre6-r0 x86 {musl} (MIT) [upgradable from: musl-1.1.24-r9]

I wonder if I need to remove musl -1.1.24? If so, how do I do that? (Sorry,
just guessing)

**Comment by tbodt** [comment]
I think `apk upgrade` would upgrade 1.1.24 to 1.2.2_pre6. Note that this is different from `apk update`. If that doesn't do anything, I'll need to look into this a bit more.

**Comment by ifuchs** [comment]
Wow, u r a genius. That worked and keybase cli runs. However, keybase requires that I not be superuser so used adduser and then su to change to the new account. If I then try to send a keybase message, it hangs ish. I suppose this could now be a keybase issue.

**Comment by ifuchs** [comment]
Not sure where the problem lies with keybase but I suppose that ish should not hang with no way to recover other than killing the app.

**Comment by tbodt** [comment]
Yep, that shouldn't happen.

**Comment by ifuchs** [comment]
Just to have another point of comparison with a fresh install of keybase, I  installed it into a docker container running alpine on a Raspberry Pi Zero W. Given the proper repositories (thanks again) I was able to install the keybase cli and it works (up to a point). I can issue commands that are hanging ish; however, after provisioning the new "device", I tried sending a message and that resulted in a segmentation fault. This may simply be due to the inability of the Rpi Zero W to handle this but it does suggest that there an ish ishue (couldn't resist) :-)

**Comment by 62f** [comment]
@ifuchs: ```dmesg```

**Comment by ifuchs** [comment]
dmesg is 1200 lines of the form: nnn syscall 383
